bower.json

```diff
@@ -15,8 +15,12 @@
   ],
   "license": "BSD",
   "ignore": [],
+  "dependencies": {
+    "es6-promise": "stefanpenner/es6-promise#^4.0.0",
+    "template": "webcomponents/template#master",
+    "webcomponents-platform": "webcomponents/webcomponents-platform#master"
+  },
   "devDependencies": {
-    "web-component-tester": "^4.0.1",
-    "polymer": "2.0-preview"
+    "web-component-tester": "6.0.0-prerelease.3"
```

Comment:
forcing this version because of https://github.com/Polymer/web-component-tester/issues/476